GenumKit/Stencil/Filters.swift

```diff
@@ -106,3 +126,83 @@ struct NumFilters {
     return "\(percent)%"
   }
 }
+
+struct ModelParsingFilters {
```

Comment:
So, this took me the longest time to work out. AFAIK, it is impossible to implement them in the template because of the limitations of Stencil. To take them one by one: